client/components/mma/upgrade/UpgradeSupportAmountForm.tsx

```diff
@@ -221,9 +223,11 @@ export const UpgradeSupportAmountForm = ({
 							/>
 						</div>
 					)}
-					{!errorMessage && (
+					{!errorMessage && !!chosenAmount && (
 						<BenefitsDisplay
-							chosenAmountDisplay={`${currencySymbol}${chosenAmount} per ${mainPlan.billingPeriod}`}
+							chosenAmountDisplay={`${currencySymbol}${formatAmount(
```

Comment:
as discussed it might be slightly better if this component worked out this inside itself instead of passed as a prop but I'll leave it up to you 🙂